tests/drivers/i2s/i2s_api/src/main.c

```diff
@@ -45,11 +45,14 @@ void test_main(void)
 			ztest_unit_test(test_i2s_rx_transfer_configure_1),
 			ztest_unit_test(test_i2s_state_not_ready_neg),
 			ztest_unit_test(test_i2s_state_ready_neg),
+#ifndef CONFIG_I2S_MCUX_FLEXCOMM
 			ztest_unit_test(test_i2s_state_running_neg),
 			ztest_unit_test(test_i2s_state_stopping_neg),
 			ztest_unit_test(test_i2s_state_error_neg));
+#endif
```

Comment:
We only need to skip test_i2s_state_running_neg

Reply:
ok